src/cpu/cgroup_v2.rs

```diff
@@ -95,30 +135,35 @@ mod test {
     #[test]
     fn test_read_and_parse_v2_sys_stat_garbage() {
         let path = Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_garbage");
-        match read_and_parse_v2_sys_stat(&path) {
+        let max_file_path = Path::new("fixtures/linux/fs/cgroup_v2/cpu.max");
+        match read_and_parse_v2_sys_stat(&path, &max_file_path) {
             Err(ProbeError::UnexpectedContent(_)) => (),
             r => panic!("Unexpected result: {:?}", r),
         }
     }
```

Comment:
```suggestion
    }
    

    #[test]
    fn test_read_and_parse_v2_sys_max_garbage() {
        let path = Path::new("fixtures/linux/sys/fs/cgroup_v2/cpu.stat_1");
        let max_file_path = Path::new("fixtures/linux/fs/cgroup_v2/cpu.max_garbage");
        match read_and_parse_v2_sys_stat(&path, &max_file_path) {
            Err(ProbeError::UnexpectedContent(_)) => (),
            r => panic!("Unexpected result: {:?}", r),
        }
    }
```

Could use a test for what happens when the max file is garbage. Would need a `cpu.max_garbage` fixture with content like `max garbage` in there.

Reply:
What do you mean?

If the first segment is `max`, it's already going to ignore it. Is it a possible scenario?